action.yaml

```
@@ -2,25 +2,48 @@ name: 'EC-validator'
 description: To run ec against a particular container image built by RHTAP
 
 inputs:
+
+
   image:
     description: The image reference
     required: true
 
   key:
     description: The public key for verifying signatures
-    required: true
+    required: false
 
   policy:
     description: A policy configuration file which determines what policies are to be applied
-    default: github.com/enterprise-contract/config//default
+    required: false
+
+  identity:
+    description: The identity used to validate the certificate associated with the image
```

Same here - if the identity param needs to be a regex, then we should explain that in this description also.

---

Same kind of thing here:

```suggestion
    description: A regexp string used to match the certificate identity associated with the image
```